common/src/main/java/com/wynntils/screens/maps/AbstractMapScreen.java

```diff
@@ -41,10 +41,6 @@
 public abstract class AbstractMapScreen extends WynntilsScreen {
     protected static final float SCREEN_SIDE_OFFSET = 10;
     private static final float BORDER_OFFSET = 6;
-
-    // Zoom is the scaling of the map. The bigger the zoom, the more detailed the map becomes.
```

Comment:
Why did you not move the comment with the fields..?

Reply:
i thought it was not really needed